scripts/generateTpcScripts.ts

```diff
@@ -13,32 +12,27 @@ export interface TpcDescriptor {
   tpcTypeImport: string
   key: string
   registry?: any
-  options: ({
```

Comment:
any particular reason to change this? this matches the registry interface so keeps code closer aligned